task/blue-green-deploy/0.1/clusterrolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
I think you need to keep this file under different directory from `task` as the kind is different. So your `clusterrolebinding.yaml` will be in a new directory following a structure as `/ClusterRoleBinding/default-blue-green/0.1/default-blue-green.yaml`

cc @sthaha  @vdemeester